app/views/pages/privacy_policy.html.erb

```diff
@@ -89,11 +93,22 @@
       We’ll also look at any feedback you share - for example, through website surveys.
     </p>
 
-    <h3 class="govuk-heading-m">Getting insight to make government policy better</h3>
+    <h3 class="govuk-heading-m">Getting insight to inform government policy</h3>
 
     <p class="govuk-body">
-      We may use your data to help inform government policy around teacher recruitment and retention.
+      Your data contributes to our understanding education, training and care in England, helping us to inform government policy.
     </p>
+    
+    <h3 class="govuk-heading-m">Identifying you so that you can access DfE services</h3>
+    
+    <p class="govuk-body">
+      If we do not already hold your data on our records, we’ll create a professional record for you. 
```

Comment:
a _professional_ record?

Reply:
e.g. the DQT

Reply:
I was debating whether to drop that bit and go straight in with the 'identifying you' stuff. Thoughts?